package.json

```diff
@@ -39,6 +39,7 @@
     "@trufflesuite/metalsmith-markdown-precompiler": "^1.0.1",
     "handlebars-helpers": "^0.10.0",
     "metalsmith-debug": "^1.2.0",
-    "metalsmith-env": "^2.1.2"
+    "metalsmith-env": "^2.1.2",
+    "win-node-env": "^0.4.0"
```

Comment:
What's win-node-env used for?

Reply:
not sure Why I did that. must be having some trouble setting the dev env. should I remove that and push the change?